ledger/src/blockstore_processor.rs

```diff
@@ -1434,7 +1434,7 @@ fn load_frozen_forks(
             let mut progress = ConfirmationProgress::new(last_entry_hash);
 
             let mut m = Measure::start("process_single_slot");
-            let bank = bank_forks.write().unwrap().insert(bank);
+            let bank = bank_forks.write().unwrap().insert_from_ledger(bank);
```

Comment:
What is this change for?

Reply:
This is to differentiate between the banks that are inserted from the ledger at startup vs the banks added during steady state. This helps bank_forks to compute the largest slot that was present in the ledger at cold start.